db/agents.go

```diff
@@ -50,3 +51,13 @@ func GetOrCreateAgent(ctx *api.Context, name string) (*models.Agent, error) {
 
 	return a, nil
 }
+
+func CreateAgent(ctx *api.Context, name string, personID *uuid.UUID, properties map[string]string) error {
+	a := models.Agent{
+		Name:       name,
+		PersonID:   personID,
+		Properties: properties,
```

Comment:
Need a type=agent

Reply:
@moshloop why have type=agent in an agents table? What would other types be?